File: cleiton031/programacao_basica/Questao09.java
import java.util.Scanner;

public class Questao09 {
    public static void main (String[] args) {
        double d;
        double ca;
        double cg;
        double ca1;
        double cg1;
        Scanner s;

        s = new Scanner(System.in);
        System.out.println("***Quantidade de Salgados***");

        System.out.print("Informe o total de dinheiro que posui:");
        d = s.nextDouble();
        System.out.print("Quanto custava o Salgado?");
        ca = s.nextDouble();
        System.out.print("Quanto custa um Salgado?");
        cg = s.nextDouble();

        ca1 = d/ca;
        cg1 = d/cg;

        System.out.println( );
        System.out.println("Antes com R$"  + d +  "Voce podia comprar"  + ca1 +  "Salgados");
        System.out.println("Agora com R$"  + d +  "Voce pode comprar"  + cg1 +  "Salgados");
    }
}

File: cleiton031/programacao_basica/questao09.py
print("$$$$ Quantidade de Salgados $$$$")

d = float(input("Informe o total de dinheiro que posui:"))
ca = float(input("Quanto custava um salgado?"))
cg = float(input("Quanto custa um salgado?"))

ca1 = (d)/(ca)
cg1 = (d)/(cg)

print( )
print("Antes com R$" + str(d) + "Voce podia comprar" + str(ca1) + "salgados")
print("Agora com R$" + str(d) + "Voce so pode comprar" + str(ca1) + "salgados")
